src/extensions/resizable/bootstrap-table-resizable.js

```diff
@@ -8,7 +8,9 @@ const isInit = that => that.$el.data('resizableColumns') !== undefined
 
 const initResizable = that => {
   if (that.options.resizable && !that.options.cardView && !isInit(that)) {
-    that.$el.resizableColumns()
+    that.$el.resizableColumns({
+      store: window.store
```

Comment:
Why this change here?

Reply:
Its to save the column resizing, which is already implemented in the [jquery-resizable](https://github.com/dobtco/jquery-resizable-columns) lib.

Reply:
Ok, thanks!